ptvsd/wrapper.py

```diff
@@ -1612,6 +1612,13 @@ def start_client(host, port, addhandlers=True):
     return pydevd
 
 
-# These are the functions pydevd invokes to get a socket to the client.
-pydevd_comm.start_server = start_server
-pydevd_comm.start_client = start_client
+def install(start_server=start_server, start_client=start_client):
```

Comment:
This neuters the assert that we have in `pydevd/__init__.py` about how pydevd must not be imported before ptvsd - now it can be imported before, but if you don't call install, the detours won't work.

Reply:
Unlike the first time I tried this, this function now forces a re-import of the "pydevd" module.  Alternately, we could manually replace `pydevd.start_server` and `pydevd.start_client`.

Also, where is `pydevd/__init__.py`?

Reply:
Sorry, that was meant to be `ptvsd/__init__.py`.

What happens if some ptvsd code accidentally imports pydevd before we so the refresh below (e.g. to use some constant from it). Do we end up with two copies of that module hanging around in memory, but no harmful effects otherwise?

Reply:
I've updated the code to monkeypatch the start functions rather than re-importing the pydevd module.  That fixes issues where pydevd was already imported.  I've also added some comments on why we have to jump through these hoops.